dipy/tracking/tests/test_streamline.py

```diff
@@ -516,6 +517,46 @@ def test_unlist_relist_streamlines():
         assert_array_equal(streamlines[i], streamlines2[i])
 
 
+def test_deform_streamlines():
+    # streamlines needs to be a list of streamlines
+    A = streamlines = np.array([[1, 2, 3], [1, 2, 3.]])
```

Comment:
Why are you allocating into streamlines if you overwrite it in the following line?

Reply:
this is a mistake,  changed.